src/ciloptions.ml

```diff
@@ -165,7 +165,17 @@ let options : (string * Arg.spec * string) list =
     "--strictcheck", Arg.Unit (fun _ -> Cilutil.doCheck := true;
                                         Cilutil.strictChecking := true),
                      " Same as --check, but treats problems as errors not warnings.";
-    "", Arg.Unit (fun _ -> ()), "";
+
+    "--cstd", Arg.String (fun s -> Cil.cstd := Cil.cstd_of_string s),
+                      " Specify the c language standard. Choose between c90, c99, c11";
+
+    "--gnu89inline", Arg.Set Cil.gnu89inline,
+                      "Use gnu89 semantic for inlining";
+
+    "--mergeinlines", Arg.Unit (fun _ -> Mergecil.merge_inlines := true),
+                      " Try to merge definitions of inline functions. They can appear in multiple
+                      files and we would like them all to be the same. This can slow down the
+                      merger an order of magnitude.";
```

Comment:
Will this string contain line-breaks and (potentially excessive) whitespace? Not sure how multi-line string literals work in OCaml

Reply:
It actually works as is in the output. But because `Arg.align` is intended for one-line descriptions only, I shortened this a little. Also, I fixed some other description strings, that were not aligned properly due to containing some `\t`. The strings are only aligned at the first occurring white space (which is used in most cases here) if there is no tab. The manual alignment of multi-line descriptions that was tried for some cases is broken and there does not seem to exist a clean approach how to achieve this with `Arg.align`. So I decided to stick with the first line being aligned and the few long descriptions simply being wrapped at the end of line.